### PR TITLE
added badrequest and versions for j unit and swagger

### PR DESCRIPTION
**Pull request by EdiTeare**

### CODE REVIEW

**Review by MahletSeyoum — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -46,7 +48,7 @@
 		<dependency>
 			<groupId>org.springdoc</groupId>
 			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
-			<version>2.0.2</version>
+			<version>${swagger.api}</version>
 		</dependency>
```
Comment (MahletSeyoum) [inline pom.xml:52]: Please change version variable name to openapi.version

File: pom.xml [hunk 2]
```diff
@@ -19,6 +19,8 @@
 		<spring.actuator>3.0.0</spring.actuator>
 		<spring.validation>3.0.0</spring.validation>
 		<spring.starter.web>3.0.0</spring.starter.web>
+		<junit.version>4.13.2</junit.version>
+		<swagger.api>2.0.2</swagger.api>
```
Comment (MahletSeyoum) [inline pom.xml]: change swagger to openapi 👎 🤬

**Review by MahletSeyoum — APPROVED** [review]
You better lose the attitude or the job! Your choice!

File: pom.xml (hunk 1 above)
Comment (EdiTeare) [reply pom.xml:52]: done 😎

**Pull request merged**